cmd/pebble/wait.go

```diff
@@ -110,16 +110,16 @@ func (wmx waitMixin) wait(id string) (*client.Change, error) {
 		}
 
 		for _, t := range chg.Tasks {
+			nowLog := lastLogStr(t.Log)
```

Comment:
This fixes a bug where it was missing logs when task.Status != Doing (eg: Done).

Reply:
This is the one thing I still need to look into -- will do tomorrow.

Reply:
For now, I've reverted this change until I understand it better. We can always fix in a subsequent PR if there's fixing that needs to happen.

Reply:
To close the loop on this, I opened https://github.com/snapcore/snapd/pull/11121 to discuss with the Snap folks. We should probably not be using `wait()` here. We could use our new Pebble "long polling" wait variant instead anyway, instead of polling every 100ms. It's not like Snap where we really need lots of progress updates.